docs/examples/01_vqe.ipynb

```diff
@@ -144,12 +144,12 @@
     "\n",
     "from qiskit_ibm_runtime import QiskitRuntimeService, Estimator, Session, Options\n",
     "\n",
-    "USE_RUNTIME_SERVICE = False\n",
-    "\n",
-    "service = None\n",
-    "if USE_RUNTIME_SERVICE:\n",
-    "    service = QiskitRuntimeService()\n",
-    "    backend = \"ibmq_qasm_simulator\"\n",
+    "service = QiskitRuntimeService(\n",
+    "    channel='ibm_quantum',\n",
+    "    instance='ibm-q/open/main',\n",
+    "    token='<Your Token>',\n",
+    "    verify=False\n",
+    ")\n",
```

Comment:
Is it not possible to run this example locally anymore?

Reply:
It may not be simple differences to make it work locally.  I'll look into it in a separate issue.

Reply:
I don't think it's a big deal, but we probably should make a note about that early in the doc (since most of the rest of the examples we have can run locally)

Reply:
The Aer simulator path is added for local execution.